src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -411,7 +413,57 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             onCommentUpdate();
         };
 
-        api.getFeedAPI(false).updateComment(file, id, text, hasMention, permissions, successCallback, errorCallback);
+        api.getFeedAPI(false).updateComment(
+            file,
+            id,
+            text,
+            undefined,
+            hasMention,
+            permissions,
+            hasReplies,
+            successCallback,
+            errorCallback,
+        );
+
+        // need to load the pending item
+        this.fetchFeedItems();
+    };
+
+    updateCommentStatus = (
```

Comment:
Does this need to be a separate handler or can we pass down `this.updateComment` to the components, instead? It looks like all the underlying logic supports either/or text/status.

Reply:
I've removed it and integrated into `updateComment` method. I've also updated the use of this handler by the `Comment` component.